public/app/features/dashboard/dashgrid/DashboardRow.tsx

```diff
@@ -112,7 +113,7 @@ export class DashboardRow extends React.Component<DashboardRowProps, any> {
             &nbsp;
           </div>
         )}
-        <div className="dashboard-row__drag grid-drag-handle" />
+        {editModeEnabled && <div className="dashboard-row__drag grid-drag-handle" />}
```

Comment:
can you rename to canEdit & use it on this 101 as well? 
https://github.com/grafana/grafana/blob/b74e76298f704d871b5edc1bfb704da6bfd7d860/public/app/features/dashboard/dashgrid/DashboardRow.tsx#L101